src/current/v26.1/ory-integration-guide.md

```diff
@@ -0,0 +1,974 @@
+
+In this example environment, both Ory and CockroachDB are within the us-east-1 region:
+
+- **CockroachDB**: One Virtual Private Cloud (VPC) in region (`us-east-1`) with three subnets, distributed across distinct availability zones. The CockroachDB cluster itself consists of three nodes, each deployed in a separate AZ to enable fault tolerance and quorum-based consistency. A NLB sits in front of the cluster to evenly route incoming requests to the appropriate database node.
+
+- **Ory**: A separate VPC in the same region (`us-east-1`), also using three subnets, each placed in a different availability zone to ensure high availability. An Amazon EKS (Elastic Kubernetes Service) cluster was deployed with three worker nodes — one in each AZ—to distribute the workload evenly.
+For the purposes of this example, the EKS cluster is publicly accessible, and the service ports are exposed via a load balancer. All Ory components — Hydra, Kratos, and Keto — are configured to connect to the CockroachDB cluster through the NLB, ensuring consistent and resilient backend access.
+
+## Set up a joint CockroachDB/Ory environment
+
+This tutorial walks you through the manual setup of a joint CockroachDB/Ory environment. 
+
+<!-- Alternatively, you can follow [the instructions at the end of this tutorial](#alternative-terraform-setup) to automate this process using the existing Ory integration github project. -->
+
+### Before you begin
+
+Before starting this tutorial, read the [Ory overview]({% link {{ page.version.version }}/ory-overview.md %}). 
+
+To complete this tutorial, you will need:
+
+- An [AWS account](https://aws.amazon.com/resources/create-account/) with permissions to create EKS clusters and EC2 resources.
+- A configured [AWS CLI profile](https://docs.aws.amazon.com/cli/v1/userguide/cli-configure-files.html).
+- [Terraform](https://developer.hashicorp.com/terraform), [kubectl](https://kubernetes.io/docs/reference/kubectl/), [eksctl](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html), and [Helm (v3+)](https://helm.sh/) installed locally.
+- Basic knowledge of [Kubernetes](https://kubernetes.io/) concepts.
+- (Optional) A domain and DNS configuration if you plan to expose services publicly.
+
+Estimated setup time: 45–60 minutes.
+
+### Step 1. Provision a CockroachDB cluster
+
+First you need to provision the CockroachDB cluster that Ory will use for its services. Choose one of the following methods to create a new CockroachDB cluster, or use an existing cluster and skip to [Step 2](#step-2-create-databases-for-ory-services).
+
+#### Create a secure cluster locally
+
+If you have the CockroachDB binary installed locally, you can manually deploy a multi-node, self-hosted CockroachDB cluster on your local machine.
+
+Learn how to [deploy a CockroachDB cluster locally]({% link {{ page.version.version }}/secure-a-cluster.md %}). Be sure to follow the instructions for creating a **secure** cluster, needed to complete Step 2.
```

Comment:
suggest changing "needed to complete Step 2" to "needed to complete this tutorial" for future ease of maintenance